### ISSUE TITLE
Unable to obtain ACME certificate caused by timeout

### ISSUE THREAD

**Issue by Haxhi03** [scafer/traefik-duckdns-letsencrypt#5]
level=error msg="Unable to obtain ACME certificate for domains \"example.duckdns.org,*.example.duckdns.org\"" routerName=websecure-traefik-docker-compose@docker rule="Host(`traefik-docker-compose`)" error="unable to generate a certificate for the domains [example.duckdns.org *.example.duckdns.org]: error: one or more domains had a problem:\n[*.example.duckdns.org] time limit exceeded: last error: read udp 172.21.0.11:45325->35.183.157.249:53: i/o timeout\n[examplemu.duckdns.org] time limit exceeded: last error: read udp 172.21.0.11:57019->35.183.157.249:53: i/o timeout\n" providerName=myresolver.acme ACME CA="https://acme-v02.api.letsencrypt.org/directory"

so basically i followed the steps and modified the .env file, but cannot grab a certificate, any ideas?

EDIT: solved by changing DNS resolver
